lib/peeps.rb

```diff
@@ -0,0 +1,36 @@
+require 'pg'
+
+class Peeps 
+
+  attr_reader :peep, :date
+
+  def self.all
+    set_environment
+    result = @connection.exec("SELECT * FROM peeps;")
+    peeps = result.map { |peep| Peeps.new(peep: peep['peep'], date: peep['date']) }
+    peeps.reverse
+  end
+
+  def self.post(peep)
+    set_environment
+    now = Time.new 
```

Comment:
Using inbuilt Class Time - good! I embedded my 'time' in the database, different approach!